.gitignore

```diff
@@ -28,6 +28,8 @@
 # Python-specific files
 __pycache__
 *.pyc
+build
+.eggs
```

Comment:
Typically we want to keep the .gitignore file the same accross all of the libraries. Did you run into some issue that couldn't be resolved without adding these here?